scipy/stats/tests/test_stats.py

```diff
@@ -1480,6 +1482,16 @@ def test_moment(self):
         y = stats.moment(self.testcase, 4)
         assert_approx_equal(y, 2.5625)
 
+        y = stats.moment(self.testcase, [1, 2, 3, 4])
+        assert_approx_equal(y[0], 0.0, 10)
+        assert_approx_equal(y[1], 1.25)
+        assert_approx_equal(y[2], 0.0)
+        assert_approx_equal(y[3], 2.5625)
```

Comment:
Maybe add a comment above this "check array_like moment", and the checks can do done at once: ``assert_allclose(y, [0, 1.25, 0, 2.5625], rtol=xxx)``